src/commands/createWebApp/createWebApp.ts

```diff
@@ -16,14 +16,15 @@ export async function createWebApp(context: IActionContext & Partial<ICreateChil
         node = <AzureParentTreeItem>await ext.tree.showTreeItemPicker(SubscriptionTreeItem.contextValue, context);
     }
     let newSite: WebAppTreeItem;
+    const authErrorRegexp = new RegExp(`The client '.+' with object id '.+' does not have authorization to perform action 'Microsoft\.Web\/serverfarms\/read' over scope '.+' or the scope is invalid\. If access was recently granted, please refresh your credentials\.`);
```

Comment:
Is this is a good approach?

I found this specific error to be present many times with the same wording. 
![image](https://user-images.githubusercontent.com/17537419/114941705-5648e480-9df8-11eb-97ff-4356d8b109f0.png)
Since this is the case, would shortening the regex be a good idea?